### ISSUE TITLE
Elemental Overload in 3.16

### ISSUE THREAD

**Issue by mojimi** [PathOfBuildingCommunity/PathOfBuilding#3345]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### What is the expected behaviour/value?

In 3.16, allocating the elemental overload keystone should give you 40% more elemental damage with hits and ailments

### What is the actual behaviour/value?

Your crit multiplier gets reduced to 100%

### How to reproduce the issue

Allocate Elemental Overload using dev fork (3.16)

Also let me say I would like very much to help on this, but couldn't find where in the code the calculation for EO happens, if someone wants to point out I will try to solve this

### Build code

```shell
eNrdXG1z2kgS_rz-FSqqrmq3Dtua0bvL3iuwscOesVlw4ni_uMbSANoICesFh6Ty369nJEDGgBvF2avdpIqA1K_P9HRPj0Y5_s_ncaBMeZz4UXhSIwdqTeGhG3l-ODypvb8537dr__l177jL0tH1oJn5gbjz695Px_K7EvApD05qDrClLB7y9MNclHYP1yYsTEc8Cjvszyi-iLyT2lUU8prywELPT-e_3IAlyRUb85ParZ-6o5rCEpeH3unyeivgYx6mLPCTtKaMmR_2I_cTTy_iKJuA3TVl6vOnTuQBbbvTve7dlJT7YVk5GP_TcTdgMx73U5YqCXyc1BqAARvydz6In7IgAzkG1al2QKmmaiY1aodbGc_YGD535O1POPcWPNqBYxsW0an81P7_TN2YtwYD7qb-lJ_Gfno6YqG79NE-2OjY7tSdLEj9SeDzeMFBNtG_eyGcqOom4psIouas21_QUtV2dIGA7Ri2Zf0YvmgZRhstg1gfNQMYlwpaBG97GPopr8jcjfwkCr_DvzLrRhdPo_GDH1bycM7amA53Ze2wkJ1GyXIIzI2kl_6APyPd6Eqrj6PrQRrAUQozuzyGTJfiGISxOzH00-VsMvSNFvPHMqFjbyI8458XVPo2cWVCQjbO43a49AHy5RaBzymNzQZOoxSqDw7KpURCtqVo90hQt0N3mcu2yn0fxjzh8bSUcLdpeM5SjC8mswnGHh_ypbs6MQ4cuo3-knN3dAHls8dSjgvRkimWvhUmQV2GySDb5K6ByVSRHKsoGeC34agaMQ1LM8xtQp4jZpgHW1XuiNctiz1EKgl5PJz1Rz4PENQS2jJLGWJCMUqe-4yya0fHW1OWlCceMbe7k5OXPdkcgxxqJDB4fKXovyFHN47-FOucYDe2RjyOshg5hDkxyuXuaJb4LpRZuajscS9zn6W1zclksV7rRFO5YpYLP1gXv76qagawol5dV23Li0GwjmUzWGnK3E9nkTdE4yuV7M7RzyYTyBQiwrB8534MOCd-qarua-R16mtY_Z-yCWYlJKYhVsGSGq3g0h-O0hCas1Utm6fiCgvelxGLklU1BkWQo1WcZ0GAWmH2P_nPKVP2EPAjRf1MnAF1PW_jaqYbPYExI9G1JogpWaLusOXyZmPzdB7z8MsMLf8ZOUpBK_SyWEQ3Wscqx0s1x4eyrRff2uNJFKfy4ikL3ESKbIeTLFVC2ZKP_cS9f8gGA9F110BFLPcMWufnrdOb9odWYUWZJRGDdR9m4wfR5uX_LjNSn8ssp7hRELBJwqFhH7AgAdm-Vwx0HzK7m6LooUksenIMNRRuN_YBRAyx6O5QJsumG0MpOmAUIYwKCzgesJvZhIsISLCYFQkTQy17VxRl3mziHJRNLW4YuMtmKMrFYgZF3Qp4ww9E8cSh1oGJkNddHD0Uwth_yFJktMlWBWWHWKyjHCyvP5GzA0dYLIhQRhQrRgxtUWtwQQlLEvSwnfEBh3BHTnuZTm7YJx4WyXKeGI_lXEuUBDLmBR8nzRlUpHMB78oWyDMCqFVxBrI9PmBZIK7_nrHAT2cihZeuXuZ7rBQuJqPoSaxucjFicicntavrXqdxmd9sBGkhRKiZm587J2yUW6cNuUKUv6UHcv_UD90g86DNLUrvwvWAPQj1NWUo9lpPoyyEUhL6gdgoFrXWm_uxIlpIFYp_OgZTCtqLIHpgAV3IlgWh44cA-oo5pTvFxcZDEgWZGJ8bPp4ELP4t88SOa7CEp6yEzM2SktoyON2XGqWyxxyytkwqEnbwlo_F7w5PmcdSdthOAdBDgeqhZIRvUuBjacxWAHFzrECBKH-iCXhpBOTpbDyOIEC6kHlTlozzWNyGGtrjIlQanse9xWJvvlO9q9OLsNsqDo2DEKIspCjlWv2Wvst9ZZeJlWplh1_KwHop2xGl5Y6it_dMtoXQ28ktmsq-rZOC9a4dJhM_Zukijb-le4vQ6PKQp4WW6n5uFYd1eBmuZSlv7jqswfzv8fU5P9a5nCsvhHmK-xuWjR2gFllXNPIXUcDHb1sYCuEBYFtI3zVvSAGKlKC8NHBZNEo6Dv8y1HqwxPphqJWEVwOtnwIwW0GTFH81aHIX5IehVpZeDTYpYStsJR1_HWyLlPvDoFvVUA2-ZWnYBuGKrr9FskUP2GkUyo3qKaBXeUzWCkGXsDLzPwvcW5byuD8Z8XjD0l1SdlmcY4JA-t3Mi6PkuUQs0GXefxbOsqCesWT0Rii_lIfFOC_-OevbNIwOtl-MXRbyfhZ_X5u4RsoOXTLwKjnzP2wiQ9cwqpwen3Nj4cy5VnAsvsn9q5uYc4XlHksOUuwZwY_yYTxhHhX4JJCNZsWm50ntK9XVuqmrpvat_lW3iGPVTaI6FvwybJuYcI_q5BuYFHlix0ontl63dF2vG_ABxKpu0ToxiGnVHc2hdR0otLphWsCr25pJ69SiqlbXiSDUqWWq8J0apG44mlHXHWrodY06GkjRVUrqlDiGXYcPldQJ0akKn6Zu1ClcpY5qAqNuOnqdUqlZg5t1w3AcUArSnbpGbMsAJ6gpaFTbBh2G-NQN6tQtR7PrJtWsugYkIMvQhGbLFppNaqlwHeSAAzpo0jQdbNQ0y4BPlVKwV7ccp25Ywl_DEjp0DTAjqgbWaqrq1AWgmmladh3YTLDHMXVbwEjUugFqbODTwVhCVHBTQKNZ4Cfccxxbwgcy5UBQommq8MQAKVQXGsBwp05MlWoAqwquEN3RTLDfAjsNS8Ctg-UOSLAtkKwbqgkW2YAU-ADma8JucE6MnAb65CDXlBSiqHT-U1xy51GjFTOo5fkp965EHBRp633vUn75aZSmk-To8PDp6elgAgEbDfhnP-AHbjQ-nIAYCM59OY32haLDBvxpNjqnjWbz7l3vJrk-S-5_uzxNAod2pteTx9_NdmPy5I_an93p0wVrTM4ezj-4f3Qu-zP38qPabIQX54T98XF2dpXQOHzf1GZnFmne9jsXajxoDRqjkfdnL7z7rXXr_TExr6fp7Go4yc6odteLp_eDG2f_qdVXW80vl71G97Tbm1w2x9mZqWZXj-fj-3eTRv_Lp2489h5nd9k5uXmfOZeDR_vplo75dTA67-03-undrdW4e_jvXX_oDT-3uvTDfWxlHxsSmMM5Msf5Wdckh6n4JeeSQJaYEA6Q9dI8exRP_9bQEWKoCDoTpp6OodMNW0PQUeo4GHm6bVoORp5GCILM0VUMKjDJUN5SxyIUQ0ccGyXPtKiBsQ_mPwZlyGCo0dB0HUVnGMTBRItjORYmWFTILQg6yzFVFCqmhgkC3QQ3MHSqrmL0QqGwUPZB9cMEge7gvNAc1EwTtQIztlDKMIMBZQGlF0qgifGD2rqF0WtAabdQc4g4GH-hGGomKvNR20bhjJvjGtAhwwAVVcSiuGg2bRuTqwBmEzO-sJwwHRR-pomKPwJJElU5LN1G6dVRuBDV0FC4OKaNq1iWaaNwBkKUXtVGZRdY_-LmpUkwdLCSRak1HVTSNR2CW2do6ppsAG3JcrUDP6ADkc2KaFHEl6so5fKeuDj_IRuYDz5_UhLOYnfUT2PR532JovGdGHfdPnB0WBrD6lVkSPGk_B1naYdNFg2coL1c9MnyQXvK0jMfepxYnhGYN1iC8KNYTlkHqmlptmNSxyyOT8mOrWikxPc-z1uxLOF97kahd8vZJArl5dKjeUEqjxgUfWaPxdDdHSm9Rq-1dwVeCYK9bhxNRtydKbcs9PZOYzaAlfSRIqyCe3zgfz5SvsYsHPIj9cD4ljegix1H-ShS9JLSSbLgEO9EPf_RzwaYH8U5gyOFqnvFkB0pzX34uydV9PjjkWLae-3xJPBdX9wle19TNkyOXNk_3nvymW-9-Ce_-K3wgHz7WTP3dfWXf4l-PuYs4Z6SP07NHxXv_ZsoaaRIVUo0UBjcWW7C5pT5RoFwulDM56-R3T9TumLJgqoezAUurHNz2L-VgG54XqL8rO07vwiDfiaquU8I_eXFw21xV9qVyDjvRF5PyFCkpHznwluchnt5H1TlFPMTgXJ_IG_Tg2h-xC4PMKU0TxfH614Q0ZdERdBuCdhynK5T3H9iE6XxMEsSBgOQT_l103wN6yobxbG98OIZVTOK0qSiQet8ecWoJg_SHVnOoUf-pGgok-g6k7Rq8OJAoG_Eplcz0qymzayOJnllgNfF0ysj8I4HY57uqmhNJJlV9LwSflAnpuL08O6CteqziFTRp1eLBVIt8vTXhwcD7K5i14VXRcdReceoYE-FHKi9psabKfnp0R2Hbz3KleLLqMKEmpKYgajkp1EZVlph3M1qM0mrFr7a7lGmV0_7FWKaVAZfwwSEjiGqYPZ3LH6QI7l9RbZtFMwKUVl9GL4jXLApWfAiZw2etAJyPdGH0CophlaGt2K6wI9nFXc0BE6k-jDoldEyqniDmS_Vg_wVkxrjLFhT2_LGFFpGuW8jt0TkW2RROPCHL94JE02mL87Rypd0AP2a8hBFAWdhseGy4TUyeVxVPsNfvkmmriHmIR_P2kkzSpLlS2rvgVxppF_82F_DMl78XyTiWDS07GHa4-J982C21EXVddpgjOLYF4cPFkejF6_nzTmNNXwLFFrC3P5I4O8hkFhzgq0lzwMUBwVeCjg-nA_E8eHq_-rzP97NVVA=
```

**Comment by LocalIdentity** [comment]
Fixed in 2.12.0

**Comment by ZzZombo** [comment]
Was it though: [Path_of_Building_31 10 2021-08_31_06](https://user-images.githubusercontent.com/10889612/139562010-10c43213-7fb5-4b9b-8b49-747dc977c774.png)? I also can't find anything regarding the keystone in the changelog for that version.
